ios/Magnet/Info.plist

```diff
@@ -45,12 +47,15 @@
 	<key>UIBackgroundModes</key>
 	<array>
 		<string>bluetooth-central</string>
+		<string>fetch</string>
+		<string>location</string>
 	</array>
 	<key>UILaunchStoryboardName</key>
 	<string>LaunchScreen</string>
 	<key>UIRequiredDeviceCapabilities</key>
 	<array>
 		<string>armv7</string>
+		<string>location-services</string>
```

Comment:
Didn't for the device to have gps, but to have location-services, meaning that ipod and ipads will still be working, but location will be restricted.